pkg/spec/storage.go

```diff
@@ -467,6 +467,15 @@ func getBindMount(args []string) (spec.Mount, error) {
 			}
 			newMount.Destination = kv[1]
 			setDest = true
+		case "relabel":
+			switch kv[1] {
```

Comment:
Need to check that `len(kv) == 2` first

Reply:
Should also have a bool to prevent this from being set more than once, like setDest above

Reply:
Fixed